### PR TITLE
Adding support for InfluxDB

### PR DESCRIPTION
**Pull request by TFyre**
Adding support for InfluxDB

### CODE REVIEW

File: micrometer-registry-influx/deployment/src/test/java/io/quarkiverse/micrometer/registry/influx/deployment/DisableDefaultRegistryTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            .overrideConfigKey("quarkus.micrometer.export.influx.default-registry", "false")
+            .overrideConfigKey("quarkus.micrometer.registry-enabled-default", "false")
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class));
+
+    @Inject
+    MeterRegistry registry;
+
+    @Test
+    public void testMeterRegistryPresent() {
+        // Stackdriver is enabled (alone, all others disabled)
```
Comment (ebullient) [inline micrometer-registry-influx/deployment/src/test/java/io/quarkiverse/micrometer/registry/influx/deployment/DisableDefaultRegistryTest.java:34]:
```suggestion
        // Influx is enabled (alone, all others disabled)
```

File: micrometer-registry-influx/deployment/src/test/java/io/quarkiverse/micrometer/registry/influx/deployment/InfluxEnabledTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            .overrideConfigKey("quarkus.micrometer.export.influx.publish", "false")
+            .overrideConfigKey("quarkus.micrometer.registry-enabled-default", "false")
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class));
+
+    @Inject
+    MeterRegistry registry;
+
+    @Test
+    public void testMeterRegistryPresent() {
+        // Stackdriver is enabled (alone, all others disabled)
```
Comment (ebullient) [inline micrometer-registry-influx/deployment/src/test/java/io/quarkiverse/micrometer/registry/influx/deployment/InfluxEnabledTest.java:35]:
```suggestion
        // Influx is enabled (alone, all others disabled)
```

**Review by ebullient — CHANGES_REQUESTED** [review]
Thanks so much for this contribution! 🍻  
Well done! 🎉

A few nit comments, but please squash your commits. ;)

File: micrometer-registry-influx/deployment/src/test/java/io/quarkiverse/micrometer/registry/influx/deployment/DisableDefaultRegistryTest.java (hunk 1 above)
Comment (TFyre) [reply micrometer-registry-influx/deployment/src/test/java/io/quarkiverse/micrometer/registry/influx/deployment/DisableDefaultRegistryTest.java:34]: them copy & pastes :D

**Review by ebullient — APPROVED** [review]

**Comment by ebullient** [comment]
@all-contributors please add @TFyre for adding code to support InfluxDB

**Pull request merged**